src/Materials/effect.ts

```diff
@@ -869,6 +890,24 @@ export class Effect implements IDisposable {
         this._engine.setTextureArray(this._samplers[channel], this._uniforms[channel], textures, channel);
     }
 
+    /**

```

Comment:
This is so annoying to see WebGPU specifics here :-( I wonder what choice we could have ? I guess WebGPUEngine could extend effect ??? a bit like we do in engine extensions.

Always thinking to our min bundle

Reply:
Code moved.